google-cloud-dependencies/pom.xml

```diff
@@ -0,0 +1,810 @@
+<?xml version="1.0"?>
```

Comment:
Should be this module explicitly called `google-cloud-bom`?

Reply:
Yeah I think that is clearer.